.github/workflows/release-experimental.yaml

```diff
@@ -75,21 +75,21 @@ jobs:
           twine check dist/*
           unzip -p dist/*.whl METADATA | grep -E '^(Name|Version):'
 
-      # - name: Release to PyPI
-      #   if: env.version != '0.0.0'
-      #   env:
-      #     TWINE_USERNAME: __token__
-      #     TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
-      #   run: uv run twine upload --skip-existing --verbose dist/*
-
-      # - name: Add entry to CHANGELOG
-      #   if: env.version != '0.0.0'
-      #   uses: ncipollo/release-action@v1
-      #   with:
-      #       token: ${{ secrets.GITHUB_TOKEN }}
-      #       prerelease: false
-      #       draft: false
-      #       tag: ${{ EXPERIMENTAL_TAG }}
-      #       bodyFile: last_changelog_entry.md
+      - name: Release to PyPI
+        if: env.version != '0.0.0'
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
+        run: uv run twine upload --skip-existing --verbose dist/*
```

Comment:
Are we not releasing this to jfrog?

Reply:
That would be the best, but users won't be able to run this in CDF notebooks right? If it's in jfrog you must have access to the artifactory. 

If we want to keep experimental only for local usage, then I can change this to publish to jfrog. Wdyt?

Reply:
My I misunderstood something, but if we release to pypi then we need risk-reviewers again. Which is at least 50% of the reason for why we are doing this in the first place.